firefox-ios/firefox-ios-tests/Tests/XCUITests/URLValidationTests.swift

```diff
@@ -25,17 +25,17 @@ class URLValidationTests: BaseTestCase {
         for i in urlTypes {
             navigator.openURL(i)
             waitUntilPageLoad()
-            XCTAssertTrue(app.otherElements.staticTexts["Mozilla"].exists, "The website was not loaded properly")
-            XCTAssertTrue(app.buttons["Menu"].exists)
-            XCTAssertEqual(app.textFields["url"].value as? String, "www.mozilla.org/en-US/")
+            mozWaitForElementToExist(app.otherElements.staticTexts["Mozilla"])
+            mozWaitForElementToExist(app.buttons["Menu"])
+            mozWaitForValueContains(app.textFields["url"], value: "www.mozilla.org/en-US/")
```

Comment:
`mozWaitForValueContains` is not exactly `XCTAssertEqual`, but from time to time there may be experiments going on that query parameters such as `/v=2` are injected. It's safer just to test for `www.mozilla.org/en-US/`. 🤔

Reply:
👍 
Would it be safe to remove the `/en-US/` part as well? These tests fail for me locally since I'm in `en-CA`. :) But I know it works for Bitrise so it's not a priory.

Reply:
The Focus versions pass locally for me since they're just checking for mozilla.org.

Reply:
@isabelrios Could we assume that the runners always have the locale `US`?